src/main.js

```diff
@@ -1,9 +1,30 @@
 // query selector variables go here 👇
-
+// Image, Title, Quotes
 var imageInput = document.querySelector('.poster-img');
 var titleInput = document.querySelector('.poster-title');
 var quotesInput = document.querySelector('.poster-quote');
-var showRandomPoster = document.querySelector('.show-random');
+
+// show random, make poster buttons, saved poster Button
```

Comment:
in our final push we will need to remove all these commented out lines, but I like it now while we work on the project!